spec/features/admin_sees_payments_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'spec_helper'
+
+feature 'admin sees payments' do
+  
+  background do
+    valid_user = object_generator(:user, full_name: "Jeff Knox", email_address: 'knoxjeffrey@outlook.com')
+    object_generator(:payment, amount: 999, user: valid_user)
+  end
+  
+  scenario "admin can see payments" do
+    admin_user = object_generator(:admin)
+    sign_in_user(admin_user)
+    
+    visit admin_payments_path
+    
+    expect(page).to have_content("£9.99")
+    expect(page).to have_content("Jeff Knox")
+    expect(page).to have_content("knoxjeffrey@outlook.com")
```

Comment:
What I do usually in my feature specs - for expect blocks like this. 

```ruby
# ...

validate_payment_details

# ...

def validate_payment_details
  expect(page).to have_content("£9.99")
  expect(page).to have_content("Jeff Knox")
  expect(page).to have_content("knoxjeffrey@outlook.com")
end
```

For me it is just chunk of code that must be validated, I just abstract it to a method.